TBS - Fixing deprecated methods doc (#2781)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md

<ApiLifecycle access="ie" /><br>
<ApiLifecycle access="Limited GA" /><br>

This guide covers the Javascript method that is deprecated from the Sign-In Widget and describes how to use the `showSignIn()` method instead.

---

**Learning outcomes**

> Understand the `showSignIn` methods that are used in the Sign-In Widget so that you can set the redirect URI based on the sign-in policies that the Administrator defines.

Reviewer: ```suggestion
Understand the `showSignIn()` method that is used in the Sign-In Widget in order to set the redirect URI based on the sign-in policies defined by the Administrator.
```

**What you need**

* [Okta Sign-In Widget that is updated to the latest available release](/docs/guides/oie-upgrade-sign-in-widget/main/)

**Sample code**

n/a

---

## JavaScript sign-in method changes

> After you upgrade your org to Okta Identity Engine, the [setCookieAndRedirect()](https://github.com/okta/okta-signin-widget#renderel) JavaScript method is deprecated from the Sign-In Widget. In the Classic Engine, your app integration calls the `setCookieAndRedirect()` method which means that your app integration sets the redirect URI. However, in the Identity Engine, your app integration shouldn't assume whether it sets the redirect URI as part of the flow. Administrators set the sign-in policies in the [Okta Admin Console](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-policies). Instead of the `setCookieAndRedirect()` method, use the `showSignIn()` method to resolve the returned promise or redirect it, based on the Administrator's policy.

Reviewer: ```suggestion
After you upgrade your org to Okta Identity Engine, the [`setCookieAndRedirect()`](https://github.com/okta/okta-signin-widget#renderel) JavaScript method is deprecated from the Sign-In Widget. In the Classic Engine, your app integration calls the `setCookieAndRedirect()` method, which means that your app integration sets the redirect URI. However, in the Identity Engine, your app integration shouldn't assume whether it sets the redirect URI as part of the flow. Administrators set the sign-in policies in the [Okta Admin Console](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-policies). Instead of the `setCookieAndRedirect()` method, use the `showSignIn()` method to resolve the returned promise or redirect it, based on the Administrator's policy.
```
